ios/CodeScannerProcessorPlugin.m

```diff
@@ -231,8 +232,8 @@ + (void)setEventEmitter:(RCTEventEmitter*)eventEmitterArg {
 + (void)load {
   [FrameProcessorPluginRegistry
       addFrameProcessorPlugin:[VisionCameraCodeScanner name]
-              withInitializer:^FrameProcessorPlugin*(NSDictionary* options) {
-                return [[CodeScannerProcessorPlugin alloc]
+              withInitializer:^FrameProcessorPlugin*(VisionCameraProxyHolder* _Nonnull proxy, NSDictionary* _Nullable options) {
+                return [[CodeScannerProcessorPlugin alloc] initWithProxy:proxy withOptions:options];
                     initWithOptions:options];
               }];
```

Comment:
This syntax is not valid...looks like you forgot to remove the extra `initWithOptions: options];`?

Reply:
Hmmm, it compiles for me (but doesn't if I get rid of `initWithOptions: options];`). [This](https://github.com/mrousavy/react-native-vision-camera/blob/b1fa06514f4f2450ca2d0387d2ec781963ed4ef8/package/ios/Frame%20Processor/FrameProcessorPlugin.h#L68) was my reference when writing this code.

Reply:
I don't know how it's compiling for you with this. You have mismatched braces/brackets...VSCode shows this:
![image](https://github.com/mgcrea/vision-camera-barcode-scanner/assets/5598961/c3c8d34a-9310-417d-bb3e-c8527f0898e7)

Should be this:
```
+ (void)load {
  [FrameProcessorPluginRegistry
      addFrameProcessorPlugin:[VisionCameraCodeScanner name]
              withInitializer:^FrameProcessorPlugin*(VisionCameraProxyHolder* _Nonnull proxy, NSDictionary* _Nullable options) {
                return [[CodeScannerProcessorPlugin alloc] initWithProxy:proxy withOptions:options];
              }];
}
```

Reply:
Oh man, that's embarrassing!

For reference, I've patch-packaged this over in a React Native project and I'm then copying it over to this repo. The code I'm running in the project has the block that you mentioned above, but I managed to screw things up when copying it over.

Good eyes - pushing up a fix!